bond: Add support of bond option `lacp_active` and `ns_ip6_target`

cathay4t opened the pull request:
Support these bond options:
 - `lacp_active`: true or false. Currently kernel only allow this option
   in `802.3ad` mode.
 - `ns_ip6_target`: list/array of IPv6 address. Currently kernel only
   support using this along with `arp_interval` > 0.

Unit test cases and integration test cases are included.
Resolves: https://issues.redhat.com/browse/RHEL-1415
Resolves: https://issues.redhat.com/browse/RHEL-85784

bengal commented:
The commit message needs to be updated (" kernel only support using this ... on `balance-rr` and `balance-xor` mode."). Otherwise, LGTM.

cathay4t committed:
bond: Add support of bond option `lacp_active` and `ns_ip6_target`

Support these bond options:
 - `lacp_active`: true or false. Currently kernel only allow this option
   in `802.3ad` mode.
 - `ns_ip6_target`: list/array of IPv6 address. Currently kernel only
   support using this along with `arp_interval` > 0.

Unit test cases and integration test cases are included.

Resolves: https://issues.redhat.com/browse/RHEL-1415
Resolves: https://issues.redhat.com/browse/RHEL-85784

Signed-off-by: Gris Ge <fge@redhat.com>

cathay4t commented:
Commit message fixed.
